Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -224,8 +647,4 @@ parameter to the `LambdaFunction` must be the event object and the event source
 
 ## Project References
 
-If you are using API Gateway event attributes, such as `RestAPI` or `HttpAPI`, you need to manually add a package reference to `Amazon.Lambda.APIGatewayEvents` in your project, otherwise the project will not compile. We do not include it by default in order to keep the `Amazon.Lambda.Annotations` library lightweight.
-
-This release only supports API Gateway Events. As we add support for other types of events, such as S3 or DynamoDB, the list of required package references will depend on the Lambda .NET attributes you are using. 
-
-**Note**: If you are using [dependency injection](#dependency-injection) to write functions for other service events , such as S3 or DynamoDB, you will need to reference these packages in your project as well
```

Comment:
I purposely deleted this `Note` because after rereading I realized it doesn't make sense. It was basically saying if you are using dependency injection in non API Gateway cases be sure to add your event package but those tasks are unrelated. 

It mattered in the API Gateway case, but not for DI, because the source generator is using the event package on behalf of the user. In the non API Gateway case the user is directly using the package.